x/dex/keeper/order_count.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/dex/types"
+)
+
+func (k Keeper) SetOrderCount(ctx sdk.Context, contractAddr string, priceDenom string, assetDenom string, direction types.PositionDirection, price sdk.Dec, count uint64) error {
+	store := prefix.NewStore(
+		ctx.KVStore(k.storeKey),
+		types.OrderCountPrefix(contractAddr, priceDenom, assetDenom, direction == types.PositionDirection_LONG),
+	)
+	key, err := price.Marshal()
+	if err != nil {
+		return err
+	}
+	value := make([]byte, 8)
+	binary.BigEndian.PutUint64(value, count)
+	store.Set(key, value)
+	return nil
+}
+
+func (k Keeper) GetOrderCount(ctx sdk.Context, contractAddr string, priceDenom string, assetDenom string, direction types.PositionDirection, price sdk.Dec) uint64 {
```

Comment:
Should we expose a query as well to expose the OrderCount given a contract, pair, direciton and price?